actionview/test/ujs/public/vendor/jquery-2.2.0.js

```diff
@@ -6433,7 +6433,7 @@ Tween.propHooks = {
 };
 
 // Support: IE9
-// Panic based approach to setting things on disconnected nodes
+// Panic-based approach to setting things on disconnected nodes
```

Comment:
This is third party code. No need to update it.

Reply:
Oh got It. Updated.